src/Symfony/Component/BrowserKit/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+6.3
+---
+
+* Methods `AbstractBrowser::click()` and `AbstractBrowser::clickLink()` will have a new `$serverParameters` argument
```

Comment:
```suggestion
* Methods `AbstractBrowser::click()` and `AbstractBrowser::clickLink()` have a new `$serverParameters` argument
```

Reply:
I did it that way to be consistent with the similar change : 

https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/BrowserKit/CHANGELOG.md#420